.github/workflows/datadog-static-analysis.yml

```diff
@@ -0,0 +1,21 @@
+on: [push]
+
+name: Datadog Static Analysis
+
+jobs:
+  static-analysis:
+    runs-on: ubuntu-latest
+    name: Datadog Static Analyzer
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Check code meets quality and security standards
+      id: datadog-static-analysis
+      uses: DataDog/datadog-static-analyzer-github-action@v1
+      with:
+        dd_api_key: ${{ secrets.DD_API_KEY }}
```

Comment:
Since we already have an API key for CI purposes, I'd like to suggest that we create keys specifically for this purpose and give them specific names.

eg: `secrets.STATIC_ANALYZER_DD_API_KEY` (or a better name).

That way, if a key leaks, we can know which one to change.

Reply:
Yes, this is what we have done on the datagod-agent. See https://github.com/DataDog/datadog-agent/pull/26627/files